updater/src/Updater/Cli.purs

```diff
@@ -34,6 +34,9 @@ command = OA.hsubparser $ fold
   , OA.command "sync-labels"
       $ OA.info syncLabels
       $ OA.progDesc "Sync issue label names, descriptions, and colors with the standard set"
+  , OA.command "sync-all-labels"
```

Comment:
Should this perhaps be an argument to `sync-labels` rather than its own command? Something like `updater sync-labels --targets purescript,purescript-contrib,purescript-web,purescript-node`?

Reply:
`sync-labels` can work on other repos, so others could clone this tool and use it themselves.

`sync-all-labels` is more designed for consistency across more than just the `contrib` libraries now. I honestly doubt we'll run it that often once the initial one is done.